download.tt

```diff
@@ -0,0 +1,339 @@
+      <li><a href="[%prefix%]/latest-nixos-minimal-i686-linux.iso">Minimal installation CD, 32-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-minimal-i686-linux.iso.sha256">SHA-256</a>)</li>
+    </ul>
+  </section>
+  <section class="tab-pane" id="nixos-virtualbox">
+    <p>This is a demo appliance for VirtualBox (in OVA format) that has
+      X11 and Plasma 5 enabled, as well as the VirtualBox guest additions. To
+      use it, download the OVA file, open VirtualBox, run “File → Import
+      Appliance” from the menu, select the OVA file, and click “Import”.
+      You can then start the virtual machine. When the KDE login screen
+      appears, you can log in as <strong>user <code>demo</code></strong>,
+      <strong>password <code>demo</code></strong>. To obtain a root shell, run
+      <code>sudo -i</code> in the KDE terminal (<code>konsole</code>).</p>
+
+    <ul>
+      <li><a href="[%prefix%]/latest-nixos-x86_64-linux.ova">VirtualBox appliance, 64-bit Intel/AMD</a> (<a
+          href="[%prefix%]/latest-nixos-x86_64-linux.ova.sha256">SHA-256</a>)</li>
+    </ul>
+  </section>
+  <section class="tab-pane" id="nixos-amazon">
+    <p>If you are an EC2 user, you can fire up a NixOS instance instantly
+      by using one of the AMIs listed below.</p>
+
+    <div class="row">
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Region</th>
+            <th>Root storage</th>
+            <th>Virtualisation</th>
+            <th>AMI</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          [% amis = JSON.Escape.json_decode(IO.All.new(nixosAmis).all).$latestNixOSSeries %]
+          [% FOREACH region IN amis.keys.sort %]
+          [% FOREACH type IN amis.$region.keys.sort; ami = amis.$region.$type %]
+          <tr>
+            <td><code>[% region %]</code></td>
+            <td>[% IF type.search("ebs"); THEN; "EBS"; ELSE; "Instance"; END; %]</td>
+            <td>[% IF type.search("hvm"); THEN; "Hardware"; ELSE; "Para"; END; %]</td>
+            <td><code>[% ami %]</code></td>
+            <td><a href="https://console.aws.amazon.com/ec2/home?region=[%region%]#launchAmi=[%ami%]"
+                class="btn btn-primary btn-mini">Launch</a></td>
+          </tr>
+          [% END %]
+          [% END %]
+        </tbody>
+      </table>
+    </div>
+
+    <p>You can create an instance using the AWS Management Console by
+      clicking one of the Launch buttons. You can also create an instance
+      from the command line. For example, to create an instance in region
+      <code>[% region = "eu-west-1"; type = "pv-ebs"; ami =
+      amis.$region.$type; region %]</code> using the EC2 API tools, just
+      run:</p>
+
+    <pre class="well">
+        $ ec2-run-instances [% ami %] --region [% region %] -k <em>my-key-pair</em></pre>
+  </section>
+  <section class="tab-pane" id="nixos-azure">
```

Comment:
this is completely outdated, so it don't make sense to review.

add a warning for now WHY this is the case and best link to an issue with status

Reply:
For now I'm leaving it as it is. I have this on my todo list to fix it for next release.

Reply:
ah i ment to say i'm going to leave the warning but i have no ticket to link to, yet